dragontts/app/providers/gemini.py

```
@@ -0,0 +1,302 @@
+        try:
+            async for response in responses:
+                if response.audio_content:
+                    chunks.append(response.audio_content)
+        finally:
+            # Cancel the gRPC stream on early abandon (see stream_synth).
+            _cancel = getattr(responses, "cancel", None)
+            if _cancel is not None:
+                try:
+                    _cancel()
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)